### PR TITLE
Updated DynamoDB Adaptor documentation

### PR DESCRIPTION
**Pull request by manishprivet**
## Reasoning 💡

The way DynamoDB was implemented in the adapters repo was changed, and the documentation needed to be updated accordingly.

## Checklist 🧢

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

## Affected issues 🎟
https://github.com/nextauthjs/adapters/pull/126

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Please see https://github.com/nextauthjs/adapters/pull/126#pullrequestreview-675822717

File: www/docs/adapters/dynamodb.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ npm install next-auth @next-auth/dynamodb-adapter@canary
 
 2. Add this adapter to your `pages/api/auth/[...nextauth].js` next-auth configuration object.
 
-You need to pass `aws-sdk` to the adapter in addition to the table name.
+You need to pass `DocumentClient` from `aws-sdk` to the adapter in addition to the table name.
```
Comment (balazsorban44) [inline www/docs/adapters/dynamodb.md:24]: Maybe say that you need to pass a `DocumentClient` instance, see you construct it with the `new` keyword?

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Could you please match the content of this PR with https://github.com/nextauthjs/adapters/pull/126/files

**Comment by manishprivet** [comment]
@balazsorban44 Apologies, it was hard to keep track of it.

**Review by balazsorban44 — COMMENTED** [review]
Thank you! :tada:

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**